stable/kong/templates/deployment.yaml

```diff
@@ -23,6 +23,10 @@ spec:
         app: {{ template "kong.name" . }}
         release: {{ .Release.Name }}
     spec:
+      initContainers:
+      - name: init-kong-database
+        image: busybox
+        command: ['sh', '-c', 'until nslookup {{ template "kong.postgresql.fullname" . }}; do echo waiting for {{ template "kong.postgresql.fullname" . }}; sleep 2; done;']
```

Comment:
This checks if the pg container is up and not if the Postgres is actually accepting connections, which means this check would work fine most of the time but not always.
A more stricter check could be: `nc -zv $KONG_PG_HOST ${KONG_PG_PORT-5432};`